core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -336,6 +336,12 @@
         public boolean enabled;
     }
 
+    /**
+     * Generate the report files for GraalVM Dashboard.
+     */
+    @ConfigItem
+    public boolean enableDashboardDump;
```

Comment:
Should we enable it for one of our ITs in the tree? I don't know if it slows down things a lot or requires a ton of memory?

Reply:
The dump file has 112M for a simple app. Do we enable reports (used_classes, etc.) for some ITs?

Reply:
OK, let's merge as is then. Looks a bit too big for our own good.